apps/condo/domains/notification/schema/Message.test.js

```diff
@@ -137,4 +147,78 @@ describe('Message', () => {
             'Variable "$data" got invalid value "invalid" at "data.sender"; Expected type "SenderFieldInput" to be an object.',
         )
     })
+
+    describe('Message constraints', () => {
+        it('checks Message constraints with non-null uniqKey and user', async () => {
+            const client = await makeClientWithRegisteredOrganization()
+            const admin = await makeLoggedInAdminClient()
+            const uniqKey = faker.datatype.uuid()
+
+            const [obj] = await createTestMessage(admin, { user: { connect: { id: client.user.id } }, uniqKey })
+
+            expect(obj.uniqKey).toMatch(uniqKey)
+            expect(obj.user.id).toMatch(client.user.id)
+
+            await expectToThrowInternalError(
+                async () => await createTestMessage(admin, { user: { connect: { id: client.user.id } }, uniqKey }),
+                DUPLICATE_CONSTRAINT_VIOLATION_ERROR_MESSAGE,
+            )
+        })
+
+        it('checks Message constraints with non-null uniqKey and nullish user', async () => {
+            const admin = await makeLoggedInAdminClient()
+            const uniqKey = faker.datatype.uuid()
+
+            const [obj] = await createTestMessage(admin, { uniqKey })
```

Comment:
you need to pass `user: null` here! to protect your tests from changes inside the `createTestMessage`

Reply:
done